images/macos/software-report/SoftwareReport.Toolcache.psm1

```diff
@@ -74,4 +82,6 @@ function Get-PowerShellModules {
             Version = $moduleVersions
         }
     }
+
+    return $output
```

Comment:
nit: do we really need a temporary variable here?

Reply:
corrected